pkg/disk/cloud.go

```diff
@@ -925,7 +924,7 @@ func clientToken(name string) string {
 // So we just assume they are not supported.
 var vaildDiskNameRegexp = regexp.MustCompile(`^[a-zA-Z][a-zA-Z0-9:_-]{1,127}$`)
 
-func createDisk(ecsClient cloud.ECSInterface, diskName, snapshotID string, diskVol *diskVolumeArgs, supportedTypes sets.Set[Category]) (string, createAttempt, error) {
+func createDisk(ecsClient cloud.ECSInterface, diskName, snapshotID string, diskVol *diskVolumeArgs, supportedTypes sets.Set[Category], selectedInstance string) (string, createAttempt, error) {
```

Comment:
Should we distinguish EED from other cloud disk types? Perhaps we shouldn't conflate it with cloud_* disks in types field

Reply:
For now, we support a mixed combination and have a plan in place for future distinction if needed.